confgenerator/fluentbit/outputs.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fluentbit
+
+import "fmt"
+
+func (o Stackdriver) Component() Component {
+	config := map[string]string{
+		// https://docs.fluentbit.io/manual/pipeline/outputs/stackdriver
+		"Name":              "stackdriver",
+		"Match_Regex":       fmt.Sprintf("^(%s)$", o.Match), // FIXME: properly escape, join from list
```

Comment:
FIXME

Reply:
This isn't a regression and I'm planning to fix in the next PR when I refactor this struct and how it's used.